src/proto/lbann.proto

```diff
@@ -1060,7 +1060,9 @@ message BatchNormalization {
   double scale_init = 2;     //default: 1.0
   double bias_init = 3;      //default: 0.0
   double epsilon = 4;        //default: 1e-5
-  string stats_aggregation = 5; // default: local
+  string stats_aggregation = 5; // default: local; deprecated
+  int64 statistics_group_size = 6;  // default: 1 (local)
+  bool global_statistics = 7;
```

Comment:
I get a bad feeling from having two counteracting options. Perhaps a better approach would be  to make a zero value correspond to local statistics and a negative value to global statistics.

Reply:
Good point. I've made this change. (The Python front-end still uses `0` for global statistics.)

Reply:
We should update the Python front-end. It's much easier to change the API before it's exposed to users.

Reply:
The Python frontend is doing the same thing as LBANN. The only hiccup is the prototext, where I would prefer the default to be local statistics, but `0` is the default value for an integer field.

Reply:
Should we change the convention everywhere so that a zero group size corresponds to local statistics? Having two different usages seems like it could breed confusion.

Reply:
I don't like that, because a group size of 1 would also correspond to local statistics.

Reply:
Looks like our code smell senses are in disagreement. In any case, I don't think this is a very big deal.